.tools/test/sqs_lambda_to_batch_fargate/consumer_stack/consumer_stack.py

```diff
@@ -26,78 +27,27 @@
 class ConsumerStack(Stack):
     def __init__(self, scope: Construct, id: str, **kwargs) -> None:
         super().__init__(scope, id, **kwargs)
-
-        #####################################################
-        ##                                                 ##
-        ##                 FANOUT COMPONENTS               ##
-        ##           (SQS, SNS, and Subscriptions)         ##
-        ##                                                 ##
-        #####################################################
-
-        # Locate Amazon Simple Notification Service (Amazon SNS) topic in the producer account.
-        fanout_topic_name = "aws-weathertop-central-sns-fanout-topic"
-        fanout_topic_arn = (
-            f"arn:aws:sns:us-east-1:{producer_account_id}:{fanout_topic_name}"
-        )
-        sns_topic = sns.Topic.from_topic_arn(
-            self, fanout_topic_name, topic_arn=fanout_topic_arn
-        )
-
-        # Define the Amazon Simple Queue Service (Amazon SQS) queue in this account.
+        resource_config = self.get_yaml_config("../config/resources.yaml")
```

Comment:
Absolute Path would work better here than hardcoded '..'

Reply:
It's relative to the directory where the CDK stack is designed to be run. It won't work from any other directory. In a use case where the file could be executed directly from any location, this would be an improvement. In the use case where the file is invoked indirectly from a fixed location within a rigid third party framework, relative pathing like this is fine.

Is there any other reason you think this would be an improvement? Open to other reasoning.

Reply:
See above.